app/app/Session.php

```diff
@@ -72,4 +72,9 @@ public function votes()
     {
         return $this->hasMany(Vote::class);
     }
+
+    public function getAgendaLinkAttribute()
```

Comment:
This is nitpicking, but in other places we’re using the term "url" over "link", so how about this?

```suggestion
    public function getAgendaUrlAttribute()
```